packages/orbit-components/src/SmartPassIllustrations/index.d.ts

```diff
@@ -21,4 +19,14 @@ export interface Props extends Common.Global {
 
 declare const SmartPassIllustration: React.FunctionComponent<Props>;
 
-export { SmartPassIllustration, SmartPassIllustration as default };
+declare const SmartPassV1: typeof SmartPassIllustration;
+declare const SmartPassV2: typeof SmartPassIllustration;
+declare const SmartPassV3: typeof SmartPassIllustration;
+declare const SmartPassV4: typeof SmartPassIllustration;
+declare const SmartPassV5: typeof SmartPassIllustration;
+
+export { SmartPassV1 };
+export { SmartPassV2 };
+export { SmartPassV3 };
+export { SmartPassV4 };
+export { SmartPassV5 };
```

Comment:
I think this should work:

```suggestion
export declare const SmartPassV1: typeof SmartPassIllustration;
export declare const SmartPassV2: typeof SmartPassIllustration;
export declare const SmartPassV3: typeof SmartPassIllustration;
export declare const SmartPassV4: typeof SmartPassIllustration;
export declare const SmartPassV5: typeof SmartPassIllustration;
```